mlrun/model.py

```diff
@@ -1068,6 +1073,14 @@ def __init__(
         self.reason = reason
         self.notifications = notifications or {}
 
+    def is_failed(self):
+        if not self.state:
+            return None
```

Comment:
Add a docstring / type hints that specify that the possible return values are Bool or None

Reply:
you got it